### ISSUE TITLE
Message says error and success

### ISSUE THREAD

**Issue by goetzk** [scaphilo/koalixcrm#28]
Hi,
As a follow on to issue #27 - 
After that I edited the salescontactpofitions  per tutorial. I received the following message:

```
* 		Unsuccessfull in updating the Prices There is no Price for this product: 1 One hour work in housethat matches the date: 2017-08-06 ,customer: 50 John Duffy Magnum and unit:hr
* 		The Quote "Quote: 105 from Contract: 51" was changed successfully.
```

The quote was changed successfully, so I'm assuming the first message is related to a linked object. I find the message a bit confusing so I can't help guess what caused it.

**Comment by scaphilo** [comment]
Dear Karl

From what you write it seems to me there are two separate issues:
1st part: I think this issue with the wrong "severity" level was already existing:  #7 and was solved in the new version of the koalixcrm 1.11.dev3.
We are currently trying to set up an automatic deployment on heroku so that we can get rid of the manual demo server.

2nd part of the issue: Unclear message when price calculation fails (user does not know what to do)
As this error message is very frequent i think the user should even be guided on how to solve it. Not just show the error message but also to send the user to the form where he can define the missing prices for the product.
I think the easiest way to solve it is to have an intermediate page with the missing product prices.

@Karl: If you have further questions I would appreciate to hear them

regards
Aaron

**Comment by scaphilo** [comment]
tested - seems to work in the new version
